### ISSUE TITLE
The processing time is quite slow when I run the pre-trained model

### ISSUE THREAD

**Issue by aquariusnk** [MaybeShewill-CV/lanenet-lane-detection#111]
Hi, I tested your pretrained model and the result was pretty good. However, the speed was quite slow, just about 0.65s per image, mean not even equal to 2fps. I am testing your model with conda python 3.5 and tensorflow 1.10.0 on Ubuntu 16.04.
"预测1张图像车道线, 共耗时: 0.64280s, 平均每张耗时: 0.64280s"
So could you help me on speeding up the processing time of your model to get close to the record on your paper?
Thanks for considering my question.

**Comment by aquariusnk** [comment]
Thank you for your quick reply. However I have one thing to ask to make clear about the problem. In the testing phase, we can use batch size larger than 1. But in the real application, we can only feed 1 image at a time, so is there any way to speed up the process rather than increasing the batch size? Or do I have to turn off all other application such as web browser (chrome) to speed up the process?

**Comment by aquariusnk** [comment]
Hi, I just check the GPU while running your model. The state and in use resource of my Titan X seems not different from the normal state (before I running the code). So could I do some extra thing to improve the speed by using more GPU computation power?

**Comment by MaybeShewill-CV** [comment]
@aquariusnk You may let the GPU warm up firstly and optimize your own feed in pipline. For example you may set up a work pool and pick up several images to feed to the model each time. All in all do not let the io problem be the bottleneck:)

**Comment by MaybeShewill-CV** [comment]
@aquariusnk I don not get your point. You may put some pics here to discuss:)

**Comment by aquariusnk** [comment]
Before running the model
![image](https://user-images.githubusercontent.com/26454564/49572075-514d0e00-f97e-11e8-9d48-7421028abdba.png)
During running the model
![image](https://user-images.githubusercontent.com/26454564/49572098-5f9b2a00-f97e-11e8-8364-5654f976fa01.png)
![image](https://user-images.githubusercontent.com/26454564/49572115-6b86ec00-f97e-11e8-8303-85e5c1569c23.png)
![image](https://user-images.githubusercontent.com/26454564/49572147-7f325280-f97e-11e8-8238-db0769045527.png)
![image](https://user-images.githubusercontent.com/26454564/49572188-9bce8a80-f97e-11e8-80ce-dd978bfbb657.png)

---------------------
My GPU did not use much resource while running the model, so is this the reason I get the slow processing time?

**Comment by MaybeShewill-CV** [comment]
@aquariusnk Maybe you are using the cpu to do the inference process. You may check your cpu utilization status:)

**Comment by aquariusnk** [comment]
Hi, I have found out the reason caused my problem and solved it. Because my Cudnn version is old, the new tensorflow-gpu version decided not to use GPU but CPU instead. After I installed the new version of Cudnn (according to this article https://medium.com/@zhanwenchen/install-cuda-and-cudnn-for-tensorflow-gpu-on-ubuntu-79306e4ac04e ), it worked well for me with the speed of 0.012 second for 1 image.
Thank you for your advice on this problem and your good work also.
